agent/src/agent/pipeline/config/handlers/factory.py

```diff
@@ -22,6 +22,7 @@ def _get_no_schema_handler(pipeline_: Pipeline, base_config: dict) -> NoSchemaCo
     handlers_protocol20 = {
         source.TYPE_CACTI: pipeline.config.handlers.cacti.CactiConfigHandler,
         source.TYPE_ELASTIC: pipeline.config.handlers.elastic.ElasticConfigHandler,
+        source.TYPE_IMPALA: pipeline.config.handlers.jdbc.JDBCConfigHandler,
```

Comment:
please add also to the _get_raw_handler
and you can also add there MSSQL

Reply:
Added